nemo/NeMo-Data-Designer/README.md

```diff
@@ -0,0 +1,25 @@
+# 🎨 NeMo Data Designer Tutorial Notebooks
+
+This directory contains the tutorial notebooks for getting started with NeMo Data Designer.
+
+## 🐳 Deploy the NeMo Data Designer microservice locally
+
+In order to run these notebooks, you must have the NeMo Data Designer microservice deployed locally via docker compose. See the [deployment guide](hhttp://docs.nvidia.com/nemo/microservices/latest/set-up/deploy-as-microservices/data-designer/docker-compose.html) for more details.
```

Comment:
There is a typo in this link 

**hhttp**://docs.nvidia.com/nemo/microservices/latest/set-up/deploy-as-microservices/data-designer/docker-compose.html